src/v/kafka/server/handlers/configs/config_response_utils.cc

```diff
@@ -1035,6 +1035,20 @@ config_response_container_t make_topic_configs(
             "Best effort target for Iceberg table lag relative to source "
             "topic, in milliseconds."),
           describe_as_string<std::chrono::milliseconds>);
+
+        add_topic_config_if_requested(
+          config_keys,
+          result,
+          topic_property_iceberg_batch_max_bytes,
+          metadata_cache.get_default_iceberg_batch_max_bytes(),
+          topic_property_iceberg_batch_max_bytes,
+          topic_properties.iceberg_batch_max_bytes,
+          include_synonyms,
+          maybe_make_documentation(
+            include_documentation,
+            "Max size, in bytes, of in-flight parquet data for an in-progress "
+            "datalake partition translator."),
```

Comment:
How does this relate to rowgroup size?

Reply:
Good question. My intuition is not  very much but I don't know the nuts and bolts particulars of how row groups get closed/flushed (or whatever the terminology is). 

As I understood it, this is more a knob related to batching in the scheduling sense - like a translation will stay open across scheduling rounds until we approach either the target lag or the "batch size", so called, at which point we flush to object storage.

Maybe it would make more sense to describe as a 'datalake' property  rather than iceberg?